### ISSUE TITLE
Issue instrumenting AZ App Service Linux application

### ISSUE THREAD

**Issue by rcskinner** [DataDog/datadog-aas-linux#10]
Experiencing issues following the documentation and provided Datadog wrapper for Azure App Services - Linux Code for Java applications. 

**Resources Used:**
1. [App Service Java Quickstart](https://learn.microsoft.com/en-us/azure/app-service/quickstart-java?tabs=javase&pivots=platform-linux-development-environment-azure-portal)
2. [Datadog App Service - Linux Code](https://docs.datadoghq.com/serverless/azure_app_services/azure_app_services_linux/?tab=java)

**Steps to Replicate:** 
1. Create Azure App Service Plan (Linux) 
2. Create Azure App Services and deploy sample code 
3. Configure Datadog environment variables 
4. Download and deploy `datadog-wrapper` 

Error Log Stream
```2023-08-14T21:50:10.922672950Z Setting Datadog environment variables
2023-08-14T21:50:10.922803062Z Creating and switching to the Datadog directory at /home/datadog
2023-08-14T21:50:10.926400078Z Adding Runtime specific dependencies
2023-08-14T21:50:10.926983130Z /tmp/startup.sh: line 87: apt-get: command not found
2023-08-14T21:50:10.927379172Z Getting the Datadog binaries
2023-08-14T21:50:10.928822287Z Downloading Datadog AAS binary https://github.com/DataDog/datadog-aas-linux/releases/download/v1.3.0/datadog-aas-v1.3.0.zip
2023-08-14T21:50:10.929217424Z /tmp/startup.sh: line 103: curl: command not found
2023-08-14T21:50:10.929603808Z Failed to download the Datadog binary succesfully.
2023-08-14T21:50:10.929905954Z Starting the appropriate Datadog bianries
2023-08-14T21:50:10.930122770Z Starting the trace agent
2023-08-14T21:50:10.931204999Z Setting up Datadog tracing for Java
2023-08-14T21:50:10.931218194Z Using version 1.4.0 of the JAVA tracer
2023-08-14T21:50:10.931223173Z Installing JAVA tracer from https://github.com/DataDog/dd-trace-java/releases/download/v1.4.0/dd-java-agent-1.4.0.jar
2023-08-14T21:50:10.931712968Z /tmp/startup.sh: line 179: curl: command not found
2023-08-14T21:50:10.931728928Z Downloading the tracer was unsuccessful
2023-08-14T21:50:10.931733336Z Removing any previous installations
2023-08-14T21:50:10.932357775Z /tmp/startup.sh: line 125: /home/datadog/v1.3.0/process_manager: No such file or directory
2023-08-14T21:50:10.942464113Z find: /home/datadog/v*: No such file or directory
2023-08-14T21:50:10.943257080Z Completed Datadog setup```

Setup fails due to a missing `curl` and `apt-get` dependency on the underlying infrastructure

**Comment by rcskinner** [comment]
Looked into this a little more. Under the hood it's running an `alpine:3.14` container which doesn't have apt-get. Was able to resolve the installation by changing the apt-gets to apks and adding curl. 
https://github.com/rcskinner/java-docs-spring-hello-world/commit/1dd6c58a352b98967a9ecda2e80d46a0636e9ff2

Not sure if this was my error or if the base image has changed and this warrants a look at the setup script!

**Comment by jcstorms1** [comment]
Hey @rcskinner sorry for the delay and thanks for letting us know. We fixed the issue in #11 and it was released in [v1.4.0](https://github.com/DataDog/datadog-aas-linux/releases/tag/v1.4.0). 

Going to close this one out.
